FE/todo-app/src/components/formComponents/Button.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+import styles from "./Components.module.css";
+import { AiFillCloseCircle } from "react-icons/ai";
+
+type Props = {
+  label?: "Create" | "Login" | "Registration";
+  buttonType?: "closeButton" | "submitType";
+  closeModal?: () => void;
+  path?: string;
+  ariaLabel?: string;
+  className?: string;
+};
```

Comment:
a co zbytek attributů z buttonu? Co když budu chtít na ten button přidat nějak aria tag? Budu ho muset pokaždé přidat? je jich třeba 40, to budu muset jit a každý si tady přidat? 

https://react-typescript-cheatsheet.netlify.app/docs/advanced/patterns_by_usecase/

Tudíž třeba ten ariaLabel bude určitě zbytečný, ten label taky, onClick taky, path už nepoužíváš ne? a co ten className?